### PR TITLE
Rearranging and renaming tests files for longevity

### PR DESCRIPTION
**Pull request by roydahan**
1. Creating another longevity-yaml with TLS SSL encryption and
authorization to be used in scylla-enterprise job.
2. Set the short longevity to run 4 days instead of 7 days and use more
compaction strategies and 2 loaders.
3. set 1TB test to more compaction strategies.

### CODE REVIEW

File: tests/longevity-1TB-7days-authorization-and-tls-ssl.yaml [hunk 1]
```diff
@@ -0,0 +1,66 @@
+test_duration: 10080
+stress_cmd: ["cassandra-stress write cl=QUORUM duration=10080m user=cassandra password=cassandra -schema 'replication(factor=3) compaction(strategy=LeveledCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..10000000 -log interval=5",
+             "cassandra-stress counter_write cl=QUORUM duration=10080m user=cassandra password=cassandra -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..10000000",
+             "cassandra-stress user profile=data_dir/cs_mv_profile.yaml ops'(insert=3,read1=1,read2=1,read3=1)' cl=QUORUM duration=10080m -port jmx=6868 -mode cql3 native -rate threads=100"]
```
Comment (larisau) [inline tests/longevity-1TB-7days-authorization-and-tls-ssl.yaml]: user profile=/tmp/cs_mv_profile.yaml

File: tests/longevity-1TB-7days.yaml [hunk 2]
```diff
@@ -1,26 +1,18 @@
...
-             "cassandra-stress counter_write cl=QUORUM duration=10080m -schema 'replication(factor=3)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..10000000",
-             "cassandra-stress user profile=/tmp/cs_mv_profile.yaml ops'(insert=3,read1=1,read2=1,read3=1)' cl=QUORUM duration=10080m -port jmx=6868 -mode cql3 native -rate threads=100"]
+stress_cmd: ["cassandra-stress write cl=QUORUM duration=10080m -schema 'replication(factor=3) compaction(strategy=LeveledCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..10000000 -log interval=5",
+             "cassandra-stress counter_write cl=QUORUM duration=10080m -schema 'replication(factor=3) compaction(strategy=DateTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..10000000",
+             "cassandra-stress user profile=data_dir/cs_mv_profile.yaml ops'(insert=3,read1=1,read2=1,read3=1)' cl=QUORUM duration=10080m -port jmx=6868 -mode cql3 native -rate threads=100"]
```
Comment (larisau) [inline tests/longevity-1TB-7days.yaml]: user profile=/tmp/cs_mv_profile.yaml

File: tests/longevity-50GB-4days.yaml [hunk 3]
```diff
@@ -0,0 +1,63 @@
+test_duration: 5760
+stress_cmd: ["cassandra-stress write cl=QUORUM duration=5760m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..100000000 -log interval=5",
+             "cassandra-stress counter_write cl=QUORUM duration=5760m -schema 'replication(factor=3) compaction(strategy=DateTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..1000000",
+             "cassandra-stress user profile=data_dir/cs_mv_profile.yaml ops'(insert=3,read1=1,read2=1,read3=1)' cl=QUORUM duration=5760m -port jmx=6868 -mode cql3 native -rate threads=100"]
```
Comment (larisau) [inline tests/longevity-50GB-4days.yaml]: the same

File: tests/longevity-1TB-7days.yaml [hunk 4]
```diff
@@ -1,26 +1,18 @@
...
+stress_cmd: ["cassandra-stress write cl=QUORUM duration=10080m -schema 'replication(factor=3) compaction(strategy=LeveledCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..10000000 -log interval=5",
+             "cassandra-stress counter_write cl=QUORUM duration=10080m -schema 'replication(factor=3) compaction(strategy=DateTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..10000000",
              "cassandra-stress user profile=/tmp/cs_mv_profile.yaml ops'(insert=3,read1=1,read2=1,read3=1)' cl=QUORUM duration=10080m -port jmx=6868 -mode cql3 native -rate threads=100"]
 n_db_nodes: 4
 n_loaders: 1
 n_monitor_nodes: 1
 nemesis_class_name: 'ChaosMonkey'
 nemesis_interval: 30
-user_prefix: 'terabyte-longevity-1-7-gce'
+user_prefix: 'longevity-1tb-7days-not-through-jenkins'
```
Comment (amoskong) [inline tests/longevity-1TB-7days.yaml]: The maximal node name is 63, the user_prefix should be less than or equal to `29`.

>>> 63 - len('-scylla-monitor-node-1faf5348-0-10')
29

Comment (amoskong) [inline tests/longevity-1TB-7days.yaml]: We have an assert in add_nodes() of GCECluster:

```
# Name must start with a lowercase letter followed by up to 63
# lowercase letters, numbers, or hyphens, and cannot end with a hyphen
assert len(name) <= 63, "Max length of instance name is 63"
```

File: tests/longevity-50GB-4days.yaml [hunk 5]
```diff
@@ -0,0 +1,63 @@
+test_duration: 5760
+stress_cmd: ["cassandra-stress write cl=QUORUM duration=5760m -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..100000000 -log interval=5",
+             "cassandra-stress counter_write cl=QUORUM duration=5760m -schema 'replication(factor=3) compaction(strategy=DateTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..1000000",
+             "cassandra-stress user profile=/tmp/cs_mv_profile.yaml ops'(insert=3,read1=1,read2=1,read3=1)' cl=QUORUM duration=5760m -port jmx=6868 -mode cql3 native -rate threads=100"]
+n_db_nodes: 6
+n_loaders: 2
+n_monitor_nodes: 1
+nemesis_class_name: 'ChaosMonkey'
+nemesis_interval: 5
+user_prefix: 'longevity-50gb-4days-not-through-jenkins'
```
Comment (amoskong) [inline tests/longevity-50GB-4days.yaml]: length issue.

File: tests/longevity-1TB-7days-authorization-and-tls-ssl.yaml [hunk 6]
```diff
@@ -0,0 +1,66 @@
+test_duration: 10080
+stress_cmd: ["cassandra-stress write cl=QUORUM duration=10080m user=cassandra password=cassandra -schema 'replication(factor=3) compaction(strategy=LeveledCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -col 'size=FIXED(1024) n=FIXED(1)' -pop seq=1..10000000 -log interval=5",
+             "cassandra-stress counter_write cl=QUORUM duration=10080m user=cassandra password=cassandra -schema 'replication(factor=3) compaction(strategy=SizeTieredCompactionStrategy)' -port jmx=6868 -mode cql3 native -rate threads=1000 -pop seq=1..10000000",
+             "cassandra-stress user profile=/tmp/cs_mv_profile.yaml ops'(insert=3,read1=1,read2=1,read3=1)' cl=QUORUM duration=10080m -port jmx=6868 -mode cql3 native -rate threads=100"]
+n_db_nodes: 4
+n_loaders: 1
+n_monitor_nodes: 1
+nemesis_class_name: 'ChaosMonkey'
+nemesis_interval: 30
+user_prefix: 'longevity-tls-ssl-1tb-7days-not-through-jenkins'
```
Comment (amoskong) [inline tests/longevity-1TB-7days-authorization-and-tls-ssl.yaml]: Length issue.

File: tests/longevity-1TB-7days.yaml (hunk 4 above)
Comment (roydahan) [inline tests/longevity-1TB-7days.yaml]: Thx Amos, I remembered the 63 chars limitations but not the fact that we are adding so many chars...

Comment (roydahan) [inline tests/longevity-1TB-7days.yaml]: I will remove some chars from my prefix, but...
Anyone object I'll fix that by removing some chars from the suffix?
For example:
-scylla-monitor-node-1faf5348-0-10
-monitor-1faf5348-0-10

Comment (amoskong) [inline tests/longevity-1TB-7days.yaml]: I thought to use  `-s-mr-1faf5348-0-10`

s -> scylla, c -> cassandra
mr -> monitor, db -> database, ld -> loader

**Comment by amoskong** [comment]
Looks good to me.

**Pull request merged**